packages/adapters/wagmi/src/client.ts

```diff
@@ -745,15 +746,6 @@ export class WagmiAdapter implements ChainAdapter {
           }
         } else if (status === 'reconnecting') {
           this.appKit?.setLoading(true)
-          const connectors = getConnectors(this.wagmiConfig)
-          const currentConnector = connectors.find(c => c.id === connector.id)
-
-          if (currentConnector) {
-            await reconnect(this.wagmiConfig, {
-              connectors: [currentConnector]
-            })
-            this.appKit?.setLoading(false)
-          }
```

Comment:
I removed this since if we're in a `reconnecting` status we don't need to reconnect again since Wagmi already does that for us [here](https://github.com/wevm/wagmi/blob/main/packages/core/src/actions/reconnect.ts#L80-L82).